tests/python_tests/utils/hugging_face.py

```diff
@@ -197,37 +235,47 @@ def convert_models(opt_model : OVModelForCausalLM,
     convert_and_save_tokenizer(hf_tokenizer, models_path)
 
 
-def download_and_convert_model(model_id: str, **tokenizer_kwargs):
-    return _download_and_convert_model(model_id, OVModelForCausalLM, **tokenizer_kwargs)
-
 @pytest.fixture(scope="module")
-def download_and_convert_embeddings_models(request):
+def download_and_convert_embeddings_models(request) -> OVConvertedModelSchema:
     model_id = request.param
     return _download_and_convert_model(model_id, OVModelForFeatureExtraction)
 
 
 @pytest.fixture()
-def download_and_convert_rerank_model(request):
+def download_and_convert_rerank_model(request) -> OVConvertedModelSchema:
     model_id = request.param
     return _download_and_convert_model(model_id, OVModelForSequenceClassification)
 
 
-def _download_and_convert_model(model_id: str, model_class: Type[OVModel], **tokenizer_kwargs):
+def download_and_convert_model(model_id: str, **tokenizer_kwargs) -> OVConvertedModelSchema:
+    return _download_and_convert_model(model_id, OVModelForCausalLM, **tokenizer_kwargs)
+
+
+def _download_and_convert_model(
+    model_id: str, 
+    model_class: Type[OVModel], 
+    **tokenizer_kwargs,
+) -> OVConvertedModelSchema:
     dir_name = str(model_id).replace(sep, "_")
```

Comment:
Using `sep` from `os.path` here is strange.

Reply:
I think the purpose is not to sanitise, just to transform `OrgId/RepoId` into `OrgId_RepoId`. Using `/` symbol directly will suffice.

Reply:
Oh no...
https://github.com/openvinotoolkit/openvino.genai/pull/2786/files#diff-8e8fb3132527bbd4b404d38e24b9f211def97e201e5ca320ea56d9a399d450e8R19